### PR TITLE
Regression: Marketplace app pricing plan description

### PR DESCRIPTION
**Pull request by tassoevan**

### CODE REVIEW

File: app/apps/client/admin/helpers.js [hunk 1]
```diff
@@ -357,15 +357,19 @@ export const appStatusSpanProps = ({
...
+	const tier = Array.isArray(pricingPlan.tiers)
+		? pricingPlan.tiers.find((tier) => tier.price === price)
+		: null;
 
 	const pricingPlanTranslationString = [
 		'Apps_Marketplace_pricingPlan',
-		pricingPlan.strategy,
-		perUser && 'perUser',
+		strategy,
+		(tier && tier.perUnit) && 'perUser',
```
Comment (sampaiodiego) [inline app/apps/client/admin/helpers.js]:
```suggestion
		tier && tier.perUnit && 'perUser',
```

Comment (sampaiodiego) [reply app/apps/client/admin/helpers.js]: I thought our lint wouldn't allow this :thinking: or I'm missing something =P

Comment (d-gubert) [reply app/apps/client/admin/helpers.js]: Either it does or `--no-verify` 😬

File: app/apps/client/admin/helpers.js [hunk 2]
```diff
@@ -357,15 +357,19 @@ export const appStatusSpanProps = ({
 export const formatPrice = (price) => `\$${ Number.parseFloat(price).toFixed(2) }`;
 
 export const formatPricingPlan = (pricingPlan) => {
-	const perUser = pricingPlan.isPerSeat && pricingPlan.tiers && pricingPlan.tiers.length;
+	const { strategy, price } = pricingPlan;
```
Comment (ggazzo) [inline app/apps/client/admin/helpers.js]: why not ?
```javascript
export const formatPricingPlan = ({ strategy, price, tiers }) =>
```
I didnt understand in some attributes we are using destructuring, in some attributes using `obj.attribute`

**Pull request merged**